sections/announcement-bar.liquid

```diff
@@ -99,32 +99,10 @@
   "class": "announcement-bar-section",
   "settings": [
     {
-      "type": "select",
+      "type": "color_scheme",
       "id": "color_scheme",
-      "options": [
-        {
-          "value": "accent-1",
-          "label": "t:sections.all.colors.accent_1.label"
-        },
-        {
-          "value": "accent-2",
-          "label": "t:sections.all.colors.accent_2.label"
-        },
-        {
-          "value": "background-1",
-          "label": "t:sections.all.colors.background_1.label"
-        },
-        {
-          "value": "background-2",
-          "label": "t:sections.all.colors.background_2.label"
-        },
-        {
-          "value": "inverse",
-          "label": "t:sections.all.colors.inverse.label"
-        }
-      ],
-      "default": "accent-1",
-      "label": "t:sections.all.colors.label"
+      "label": "t:sections.all.colors.label",
+      "default": "accent-1"
```

Comment:
NB: It's interesting that we set a `default` value for this setting, since we have no way of knowing if that scheme still exists in `settings_data.json`. I guess it makes sense for fresh installs (because the theme dev controls the default color schemes) but not always for adding new sections.